XenAdmin/Wizards/PatchingWizard/PatchingWizard_SelectServers.cs

```diff
@@ -518,9 +518,9 @@ public List<Host> SelectedServers
 
                     if (WizardMode != WizardMode.SingleUpdate)   
                         //prechecks will fail in automated updates mode if one of the hosts is unreachable
-                        return SelectedPools.SelectMany(p => p.Connection.Cache.Hosts.Where(host => enabledHosts.Contains(host))).ToList();
+                        return SelectedPools.SelectMany(p => p.Connection.Cache.Hosts.OrderBy(host => host).Where(host => enabledHosts.Contains(host))).ToList();

```

Comment:
The OrderBy() can be moved after the Where() filter because theoretically we have fewer objects to sort.
Same goes for the line below.

Reply:
Clever!  👍